Assets/Scripts/Util/Ladder.cs

```diff
@@ -23,11 +23,12 @@ private void AdaptTexture()
         var sizeDelta = ladderBottom.transform.localScale.y + ladderTop.transform.localScale.y;
         
         var spriteRenderer = texture.GetComponent<SpriteRenderer>();
-        spriteRenderer.size = new Vector2(1, height);
+        spriteRenderer.size = new Vector2(1, height - 0.5f);
```

Comment:
In some places this causes the ladder to not quite reach the ground - especially the final two ladders at the top of the scene. If it's purely height, they'll reach fully to the top/bottom of the ladder, and depending on where the ladder top is set, they'll peek a little bit above the previous floor (which looks good I think)

I'd personally revert this to just 
```suggestion
        spriteRenderer.size = new Vector2(1, height);
```